common/math/exp.go

```diff
@@ -0,0 +1,34 @@
+package math
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// Perverted constants straight out of big.Int
+	// https://golang.org/src/math/big/arith.go
+	_m    = ^big.Word(0)
+	_logS = _m>>8&1 + _m>>16&1 + _m>>32&1
+	_S    = 1 << _logS
+	_W    = _S << 3 // word size in bits
```

Comment:
Yes, but I'd prefer a const calculation package math. There is no downside and importing unsafe makes the code less portable (e.g. makes compiling to JS impossible).